pom.xml

```diff
@@ -3,17 +3,15 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.73</version>
+    <version>4.78</version>
+    <relativePath />
   </parent>
 
   <properties>
-    <!--
-    If your are using the Plugin Parent POM version 2.3 or later, just set the jenkins.version property to the version you need to depend on
-    Source: https://wiki.jenkins-ci.org/display/JENKINS/Choosing+Jenkins+version+to+build+against
-    -->
-    <jenkins.version>2.387.3</jenkins.version>
+    <jenkins.version>2.426.3</jenkins.version>
```

Comment:
Complies with the baseline standards of https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/#currently-recommended-versions, 2.387.3 has been rotated out of the update center already.